docs/about/team/the-software-development-team.md

```diff
@@ -4,6 +4,25 @@ sidebar_position: 2
 
 # The Software development team
 
+### [Maxence Guindon](https://www.linkedin.com/in/maxenceguindon)
+
+Hello! My name is Maxence Guindon, and I am a backend developer at Canadian Food
+Inspection Agency AI-Lab.
+
+I’m working on Nachet. It is a tools that will help analysts in our science labs
+to identify seeds in an image using an AI model. More specifically, my role in
+the backend is to make sure that the frontend, the database and the AI model are
+able to talk to each other. How would you ask? By building efficient API and
```

Comment:
"But how? " you might ask. By building efficient APIs and routes [...]

Reply:
Done